CHANGELOG.md

```diff
@@ -34,6 +34,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - `getMessageGranules()` - Get the granules from a Cumulus execution message, if any.
   - Added `@cumulus/common/cloudwatch-event/isFailedSfStatus()` to determine if a Step Function status from a Cloudwatch event is a failed status
 
+- **CUMULUS-639**
+  - Adds SAML JWT and launchpad token authentication to Cumulus API (configurable)
+    - **NOTE** to authenticate with Launchpad ensure your launchpad user_id is in the `<prefix>-UsersTable`
+    - when Cumulus configured to protect API via Launchpad:
+         - New endpoints
+            - `GET /saml/login` - starting point for SAML SSO creates the login request url and redirects to the SAML Identidy Provider Service (IDP)
+            - `POST /saml/auth` - SAML Assertion Consumer Service.  POST receiver from SAML IDP.  Validates response, logs the user in, and returnes a SAML-based JWT.
+         - Disabled endpoints
+            - `GET /token`
+            - `POST /refresh`
+            - `DELETE /token/:token`
+            - `DELETE /tokenDelete/:token`
```

Comment:
This endpoint isn't disabled now, I think?

Reply:
Yup.

Reply:
fc740a661